src/client/components/pages/entity-revisions.js

```diff
@@ -18,28 +18,38 @@
  */
 
 import * as bootstrap from 'react-bootstrap';
-import * as utilsHelper from '../../helpers/utils';
+import PagerElement from './parts/pager';
 import PropTypes from 'prop-types';
 import React from 'react';
+import RevisionsTable from './parts/revisions-table';
 
 
-const {Col, ListGroup, ListGroupItem, Row} = bootstrap;
-const {formatDate, isWithinDayFromNow} = utilsHelper;
+const {Col, Row} = bootstrap;
 
 /**
  * The class is derived from the React Component base class and
  * renders the 'Entity RevisionsPage' page.
  */
 class EntityRevisions extends React.Component {
-	/**
-	 * Binds the class methods to their respective data.
-	 * @constructor
-	 * @param {object} props - Properties passed to the component
-	 */
 	constructor(props) {
 		super(props);
+		this.state = {
+			results: this.props.revisions
+		};
+
+		// React does not autobind non-React class methods
+		/**
+		 * Binds the class methods to their respective data.
+		 * @constructor
+		 * @param {object} props - Properties passed to the component
+		 */
```

Comment:
This should be back right above the constructor definition.
It's a JSDocs comment, and they should be above the function definitions.